cg/meta/compress/compress.py

```diff
@@ -166,8 +166,10 @@ def clean_fastq(self, sample_id: str, archive_location: str) -> bool:
         all_cleaned: bool = True
         for run_name in sample_fastq:
             compression: CompressionData = sample_fastq[run_name]["compression_data"]
+            fastq_first: File = sample_fastq[run_name]["hk_first"]
+            fastq_second: File = sample_fastq[run_name]["hk_second"]
```

Comment:
What is the difference between these and compression.fastq_first?

Reply:
No idea, but I would not want to change the logic, just wanted to extract it to make the code a bit easier on the eye.

Reply:
Ok, let's not touch too much here